scripts/en2ja.json

```diff
@@ -0,0 +1,104 @@
+{
+  "title": "English-to-Japanese string replacement",
+  "description": "Replace EN string 'from' with associated JA string 'to'.",
+  "pairs": [
+    {
+      "from": "rel=\"canonical\" href=\"https://help.okta.com/asa/en-us",
+      "to": "rel=\"canonical\" href=\"https://help.okta.com/asa/ja-jp",
+      "note": "Many (but not all) topics contain a canonical link in the header."
+    },
+    {
+      "from": "rel=\"canonical\" href=\"https://help.okta.com/en-us",
+      "to": "rel=\"canonical\" href=\"https://help.okta.com/ja-jp",
+      "note": "Many (but not all) topics contain a canonical link in the header."
+    },
+    {
+      "from": "rel=\"canonical\" href=\"https://help.okta.com/eu/en-us",
+      "to": "rel=\"canonical\" href=\"https://help.okta.com/eu/ja-jp",
+      "note": "Many (but not all) topics contain a canonical link in the header."
+    },
+    {
+      "from": "rel=\"canonical\" href=\"https://help.okta.com/oag/en-us",
+      "to": "rel=\"canonical\" href=\"https://help.okta.com/oag/ja-jp",
+      "note": "Many (but not all) topics contain a canonical link in the header."
+    },
+    {
+      "from": "rel=\"canonical\" href=\"https://help.okta.com/oie/en-us",
+      "to": "rel=\"canonical\" href=\"https://help.okta.com/oie/ja-jp",
+      "note": "Many (but not all) topics contain a canonical link in the header."
+    },
+    {
+      "from": "rel=\"canonical\" href=\"https://help.okta.com/wf/en-us",
+      "to": "rel=\"canonical\" href=\"https://help.okta.com/wf/ja-jp",
+      "note": "Many (but not all) topics contain a canonical link in the header."
+    },
+    {
+      "from": "\"https://help.okta.com/en-us/Content/index.htm\"",
+      "to": "\"https://help.okta.com/ja-jp/Content/index.htm\"",
+      "note": "This link is in the top-level menu bar in all topics."
+    },
+    {
+      "from": "\"https://help.okta.com/en-us/Content/Topics/ReleaseNotes/okta-relnotes.htm\"",
+      "to": "\"https://help.okta.com/ja-jp/Content/Topics/ReleaseNotes/okta-relnotes.htm\"",
```

Comment:
I see there are a few replace "pairs" for each of the products and release notes. Any reason we can't just simply replace `/en-us` to `/ja-jp` everywhere?

Reply:
I've investigated the entire repo. We have links in release notes-related content that point users to EN (thus "en-us/" in their paths), we need to keep these, but as relative paths and not absolute. I've changed these all to relative paths. So, we can now remove the `Content/Topics/ReleaseNotes/*` portion, we do generally want to replace any and all instances of absolute paths that begin with `https://help.okta.com/*/en-us/`.

Reply:
I checked the existing pages in JA dirs, and also EN source. We can safely replace all instances of absolute URLs to EN topics, not just canonicals.